src/SnapshotTestCase.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Spatie\Snapshots;
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Class SnapshotTestCase.
+ *
+ * Special test case, supported Windows OS.
+ *
+ * @see {https://docs.microsoft.com/en-us/windows/desktop/fileio/naming-a-file}
+ */
+abstract class SnapshotTestCase extends TestCase
+{
+    const WINDOWS_FILE_NAME_LENGTH_LIMIT = 255;
```

Comment:
No need to tie this functionality to Windows, please rename this to `FILE_NAME_LENGTH_LIMIT`. This should be configurable.

Change this property to `protected static $fileNameLengthLimit`, and add a `public static fileNameLengthLimit(?int $limit)` function. By default, the limit should be `null`, and the check should be skipped.